plugin/src/main/java/at/helpch/chatchat/format/ConsoleFormat.java

```diff
@@ -0,0 +1,57 @@
+package at.helpch.chatchat.format;
+
+import at.helpch.chatchat.api.Format;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+
+public class ConsoleFormat implements Format {
```

Comment:
by the looks of it, this is the same as PMFormat, if so, cant we make this into a more generic class, like BasicFormat or smth? as both don't need priorities

Reply:
I did think about it but I just didn't have time to do it so I said I'm going to do it in another PR. I just noticed I forgot to annotate the class tho. I don't even know if that's needed